### ISSUE TITLE
[Enhancement / Request] Autoplay / Continue-play

### ISSUE THREAD

**Issue by FlyingGoats** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#604]
The now-defunct Rdio had an awesome feature called autoplay.

Essentially, when enabled, when your current queue finished playing, a radio station based on the last played queue would automatically start. Since GPM is good at basing stations around your thumbs up/down, I don't see why this couldn't happen here. It was great for keeping you in the "flow" should your queue end and also for general music discovery.

```
IF(last song in queue is finished playing AND autoplay is enabled AND we aren't on repeat)
THEN start radio station based on current queue/album/song
```

In terms of implementation, is it possible to start a station from current queue under the hood? If not, could it be converted to a playlist on the fly, then start the station from that, then delete playlist (unbeknownst to user)? If neither of those are possible perhaps it could just start a station based on the last song (or artist/album).

Cheers as always :+1:

**Comment by st33ly** [comment]
I think this might be a limitation of Google play music as he is pretty much just rendering the website. I would love this feature to become a reality (especially save playlist on close / auto resume on start up) but I'm not sure it can be implemented. Maybe one day Google will listen...

**Comment by MarshallOfSound** [comment]
OK so I originally typed out a massive response to this issue, but then my laptop died so know you get this tiny response :+1: :laughing: 

Although everything you request is technically feasible.  The hackiness required to make it work (simulating clicks on a bunch of DOM elements) is ridiculous and not something I really want to push when this "feature" has a very limited use case.  Especially when most people that listen to songs continuously either have there own playlists on shuffle+repeat or listen to google radio playlists which never end.

So TLDR: possible but marking as wontfix
